### PR TITLE
[MC-17] Create Date Component Input

### LINKED ISSUE #17: [Feature] - Create Date Input Component
**Description**
I want a date input box to be created that will be used as the default for the whole site, following the prototype design.
The date box must have:
- Icon at the end of the password box
- Open the date picker by clicking the icon

**Additional context**
image
image

**Acceptance Criteria**
- Separate CSS files
- The code must be responsive
- Build successfully
- lint successfully
- Use color in variables
- Open date picker select

### PR DESCRIPTION
**Pull request by pedroemanuel42**
# Feature/Issue Description
This PR added a date input component behind password input component.

# Changes Summary
- Added new input date;
- Created a new components CSS to standard configs;
- Created a new input date CSS;
- Refactored input password CSS;

# How Has This Been Tested?
To test this new component, i went into localhost and checked the input.

# Images (Opcional)
image
image
image
image

### CODE REVIEW

**Review by EvertonDuarteGuimaraes — DISMISSED** [review]

**Review by tvps20 — COMMENTED** [review]
Why is this component not aligned with the one above?
image

File: src/pages/components/index.html [hunk 1]
```diff
@@ -15,13 +15,16 @@ <h3>Inputs</h3>
...
-            <input type="password" class="form-control password-input" placeholder="Senha" required />
-            <i class="fa-solid fa-eye-slash eye-icon"></i>
+    <div class="row-password-date-input">
+        <div class="password-box" id="password">
+            <i class="fa fa-lock icon icon-lock"></i>
+            <label class="standard-label">Senha</label>
+            <input type="password" class="form-control password-input standard-input" placeholder="Senha" required="required"/>
+            <i class="fa fa-eye-slash icon icon-eye"></i>
+        </div>
+        <div class="date-box" id="date">
```
Comment (tvps20) [inline src/pages/components/index.html:28]: Why the date-box is in the div:row-password?

**Review by tvps20 — DISMISSED** [review]

File: src/assets/scss/input-date.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    
+        &::-webkit-calendar-picker-indicator {
+            filter: invert(1);
+            position: relative;
+            right: 10px;
+        }
+    }
+}
+
+.next-game {
```
Comment (feelsantiago) [inline src/assets/scss/input-date.scss]: Why is a `next-game` to a date component?

File: src/pages/components/style.scss [hunk 3]
```diff
@@ -39,6 +28,14 @@
     }
 }
 
+.row-password-input, .row-date-input {
+    background-color: $primary-dark;
+    padding: 10px;
+    //Space to put away from other components (optional)
```
Comment (feelsantiago) [inline src/pages/components/style.scss]: If a css is optional it should be a new class so the user can decide if want to add or not.

**Review by feelsantiago — CHANGES_REQUESTED** [review]

**Review by RodrigoSoaresNascimento — DISMISSED** [review]
Great!

**Review by feelsantiago — APPROVED** [review]

**Review by tvSantiago — APPROVED** [review]

**Review by tvps20 — APPROVED** [review]

**Pull request merged**